### ISSUE TITLE
Discrepancy in Reproduced Error for Geo-FNO Model on Airfoil Dataset

### ISSUE THREAD

**Issue by srinathdama** [zongyi-li/Geo-FNO#10]
Thank you for making the code and data accessible to the public!

I'm in the process of assessing FNO models with the airfoil and pipe examples and have successfully replicated the errors listed in Table 2 of your paper for both examples. However, I encountered a discrepancy with the Geo-FNO model on the airfoil dataset; the paper reports an error of 0.0138, but I observed an error of 0.01909. Could you please inform me about the PyTorch version used to obtain the results mentioned in your paper? For your reference, I am currently using PyTorch version 2.1.1 (with CUDA 11.8).

Thank you!

**Comment by srinathdama** [comment]
nvm! By increasing the learning rate to 0.002, I was able to reproduce the errors reported in the paper :)

**Comment by srinathdama** [comment]
nvm! By increasing the learning rate to 0.002, I was able to reproduce the errors reported in the paper :)
